lib/vsc/utils/fancylogger.py

```diff
@@ -443,8 +452,18 @@ def logToScreen(enable=True, handler=None, name=None, stdout=False):
 
     by default, logToScreen will log to stderr; logging to stdout instead can be done
     by setting the 'stdout' parameter to True
+
+    The `color` parameter enables or disables log colorization using
```

Comment:
better to add this piece of costring to the function, and say that the `color` parameter is passed to that function.